### PR TITLE
Added my keymap for Rorschach keyboard; minor updates to my Kinesis files.

### PR DESCRIPTION
**Pull request by Salt-Peanuts**
Added my keymap for Rorschach keyboard; minor updates to my Kinesis files.

### CODE REVIEW

File: keyboards/rorschach/keymaps/insertsnideremarks/config.h [hunk 1]
```diff
@@ -0,0 +1,30 @@
+
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+
+#include "config_common.h"
```
Comment (drashna) [inline keyboards/rorschach/keymaps/insertsnideremarks/config.h]: This line isn't actually needed.

File: keyboards/rorschach/keymaps/insertsnideremarks/config.h [hunk 2]
```diff
@@ -0,0 +1,30 @@
+
+#ifndef CONFIG_USER_H
```
Comment (drashna) [inline keyboards/rorschach/keymaps/insertsnideremarks/config.h]: If you could remove the ifndef and define blocks (as well as the endif at the end) and replace it with `#pragma once`?

File: keyboards/rorschach/keymaps/insertsnideremarks/rules.mk [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+MIDI_ENABLE = no            # MIDI controls
+AUDIO_ENABLE = no           # Audio output on port C6
+UNICODE_ENABLE = yes        # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight.  Do not enable this with audio at the same time.
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+TAP_DANCE_ENABLE = yes      # Enable Tap Dancing function
+
+ifndef QUANTUM_DIR
```
Comment (drashna) [inline keyboards/rorschach/keymaps/insertsnideremarks/rules.mk]: This ifndef block isn't needed, and is a holdover from the old makefile system.

**Comment by Salt-Peanuts** [comment]
Changes have been added.

**Comment by drashna** [comment]
Thanks!

**Pull request merged**